docassemble/clcsa-01-service-directory/data/questions/main.yml

```diff
@@ -20,17 +20,94 @@ metadata:
       <div style=" display: flex; height: 100%; align-items: center; line-height: 18px;" class="flindersAttribution">
-        Developed by Flinders University in collaboration with Community Legal Centre South Australia (CLCSA)
+        Developed by Flinders University in collaboration with Community Legal Centres South Australia (CLCSA)
       </div>
       <div style="height: 40px; display: flex; margin-left: 15px;" class="flindersLogo">
         <img src="https://staff.flinders.edu.au/etc.clientlibs/flinders/clientlibs/clientlib-site/resources/images/flinderuni_main_logo_black.png" style="height: 100%;">
       </div>
     </div>
-  exit url: https://www.clcsa.org.au/
-  exit label: Exit to CLCSA Website
+  exit label: ""
+  navigation bar html: |
+    <!-- Adds title to logo -->
+    <span id="apptitle" class="nav navbar-nav d-none d-md-block">CLCSA Service Directory</span>
+    <script>
+      apptitle = document.getElementById('apptitle');
+      nbc = document.getElementById('danavbar-collapse');
+      nbc.insertAdjacentElement('beforeBegin', apptitle);
+    </script>
+
+    <!-- Adds Safe Exit button  -->
+    <!-- Based on https://docassemble.slack.com/files/UHCKKPY3D/F013TPA5000/escape.yml -->
+    <a id="escape" href="https://www.google.com" class="btn btn-warning btn-da">Safe Exit</a>
+    <script>
+      var doc = document;  // Need space!
+
+      var outer = outer || doc.createElement( 'div' );
+      var inner = inner || doc.createElement( 'div' );
+      outer.appendChild( inner );
+
+      outer.setAttribute( 'class', 'escape-outer' );
+      inner.setAttribute( 'class', 'escape-inner' );
+
+      // Give a unique id to each escaper so we can add only
+      // one per outer/inner pair
+      var suffix = suffix || 1;  // Use `suffix` if it already exists
+      var unique_id = 'escaper_' + suffix;
+      var escaper = doc.getElementById( 'escape' );
+      escaper.setAttribute( 'id', unique_id );
+      inner.appendChild( escaper );
+      
+      if (suffix !== 1) {
+        escaper.classList.add( 'escape-overlap' );
+      }
+      suffix += 1;
+      
+      // Add to navbar
+      var after = doc.getElementById('danavbar-collapse');
+      after.insertAdjacentElement('afterEnd', inner);
+    </script>
+    
+    <style>
+      .escape-inner {
+        position: relative;
+        margin-left: .4em;
+      }
+    
+      #escaper_2, .escape-overlap {
+        position: absolute;
+        top: 0;
+        left: 0;
+      }
+      
+      .escape {
+        margin: 0;
+      }
+      
+      @media only screen and (max-width: 30em) {
+        .navbar-brand {
+          margin-right: .3rem;
+        }
+        
+        #dapagetitle {
+          margin-right: .4em;
+        }
+        
+        #dapagetitle .ma_icon {
+          margin-right: 0;
+        }
+        
+        .escape-inner {
+          margin-left: .6em;
+        }
+        
+        #dahelptoggle {
+          padding: 0;
+        }
+      }
+    </style>
```

Comment:
I'm adding to `f2ddd3f2.js` (why is it called that?) and deleting the code from `navigation bar html` as suggested.

Reply:
> I'm adding to `f2ddd3f2.js` (why is it called that?) and deleting the code from `navigation bar html` as suggested.

@mferrare It is called that because it was auto-generated (with a hash). I couldn't bear writing HTML, CSS, and JS inside yaml, so I set up a proper development environment with Vite, and it bundles my JS and CSS. I copied those output files to the `Static` folder. This allows me to write TS, use modules, SASS, Tailwindcss, etc, without adding them to DA - all necessary code would be bundled in a single file whether .js or .css.